doc/examples/filters/plot_enhancement_measure.py

```diff
@@ -0,0 +1,69 @@
+"""
+=========================================
+Quantitative Measure of Image Enhancement
+=========================================
+
+In this example, we will quantify the effect of applying contrast enhancement
+to an image using a measure called EME [1]_.
+
+It is defined as the log of the ratio of local maximum to minimum
+<<<<<<< HEAD
+within a sliding window. The code allows the user to choose the window size.
+This function was defined to quantify improvement of the image
```

Comment:
How about: This method was developed to quantify improvement of the image after processing.